ice-server/src/IceServer.cpp

```diff
@@ -36,7 +36,7 @@ IceServer::IceServer(int argc, char* argv[]) :
 
     // set processPacket as the verified packet callback for the udt::Socket
     using std::placeholders::_1;
```

Comment:
don't need the placeholders without the bind (unless I'm missing something below that uses it)